src/remotepublish.cmd.js

```diff
@@ -237,20 +243,26 @@ class RemotePublishCommand extends AbstractCommand {
   }
 
   purgeFastly() {
-    if (this._dryRun) {
+    if (this._dryRun || !(this._purge === 'soft' || this._purge === 'hard')) {
```

Comment:
```suggestion
    if (this._dryRun || this._purge === 'skip') {
```

Reply:
Your code is easier, but it use the hard purge as the default fallback in case the publish command is ever called with an invalid state. I prefer the non-destructive default.

Reply:
but the condition is very simple: either it's a dry-run or purging is skipped, then skip.
whether or not the purge is `soft` or  `hard`, doesn't matter in this case.

if you later add a new option: `--purge fluffy`, you need to update this condition.